### PR TITLE
Support keras Cropping layer for opset 9 and 10

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/ke2onnx/crop.py [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 ###############################################################################
-from ..common.onnx_ops import apply_transpose
+import numpy as np
+import keras
+from ..common.onnx_ops import apply_transpose, apply_identity, OnnxOperatorBuilder
 from .common import get_permutation_config
 
 
-def convert_keras_crop(scope, operator, container, n_dims):
+def convert_keras_crop_1(scope, operator, container, n_dims):
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/crop.py]: _1 or _v1?

File: keras2onnx/ke2onnx/crop.py [hunk 2]
```diff
@@ -50,6 +52,69 @@ def convert_keras_crop(scope, operator, container, n_dims):
...
+        else:
+            start_border = start_border + [0]
+            end_border = end_border + [0]
+
+    start_v = [0] + start_border
+    end_v = [np.iinfo(np.int32).max] + list(np.array(ori_shape) - np.array(end_border))
+
+    oopb = OnnxOperatorBuilder(container, scope)
+
+    cropped_tensor_name = oopb.add_node('Slice' if container.target_opset >= 10 else 'DynamicSlice',
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/crop.py]: no apply_slice?

File: keras2onnx/ke2onnx/crop.py [hunk 3]
```diff
@@ -50,6 +52,69 @@ def convert_keras_crop(scope, operator, container, n_dims):
...
+                                        [operator.inputs[0].full_name,
+                                         ('_start', oopb.int64, np.array(start_v, dtype='int64')),
+                                         ('_end', oopb.int64, np.array(end_v, dtype='int64')),
+                                         ('_axes', oopb.int64, np.array(axes_v, dtype='int64'))
+                                         ],
+                                        operator.inputs[0].full_name + '_cropping')
+    apply_identity(scope, cropped_tensor_name, operator.outputs[0].full_name, container)
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/crop.py]: add_node_with_output can avoid this identity node.

File: keras2onnx/ke2onnx/crop.py (hunk 1 above)
Comment (jiafatom) [reply keras2onnx/ke2onnx/crop.py]: will change to _v1

**Pull request merged**